recipes/hive_metastore.rb

```diff
@@ -40,6 +40,18 @@
   end
 end
 
+derby_dir =
+  if hive_sql == 'derby'
+    node['hive']['hive_site']['javax.jdo.option.ConnectionURL'][2].split(';').find { |o| /^databaseName/ =~ o }
+  end
+
+if derby_dir
+  directory ::File.dirname(derby_dir.split('=')[1]) do
+    action :create
+    not_if { derby_dir.nil? } # sssh, foodcritic
```

Comment:
Yes, but foodcritic complains. Using just a guard allows a null pointer. I think this needs a little more work.